tests/fixtures/bar-foos-bar-route.js

```diff
@@ -0,0 +1,6 @@
+Router.map(function() {
```

Comment:
Will do. I did find that order is relevant. The bug did not occur if the top-level route was declared *after* the nested route.